src/governance/governance.cpp

```diff
@@ -393,12 +392,12 @@ void CGovernanceManager::UpdateCachesAndClean()
 
         int64_t nTimeSinceDeletion = nNow - pObj->GetDeletionTime();
 
-        LogPrint(BCLog::GOBJECT, "CGovernanceManager::UpdateCachesAndClean -- Checking object for deletion: %s, deletion time = %d, time since deletion = %d, delete flag = %d, expired flag = %d\n",
-            strHash, pObj->GetDeletionTime(), nTimeSinceDeletion, pObj->IsSetCachedDelete(), pObj->IsSetExpired());
+        LogPrint(BCLog::GOBJECT, "CGovernanceManager::%s -- Checking object for deletion: %s, deletion time = %d, time since deletion = %d, delete flag = %d, expired flag = %d\n",
+            __func__, strHash, pObj->GetDeletionTime(), nTimeSinceDeletion, pObj->IsSetCachedDelete(), pObj->IsSetExpired());
```

Comment:
nit: After https://github.com/dashpay/dash/pull/5974/commits/dca7ae20eb57be0656e247dbb2701034488086cc get merged no need more `__func__` here.

So far as function is not actually removed and it still has a correct name, consider removing this refactoring from PR.

Reply:
These changes are introduced in https://github.com/dashpay/dash/pull/5966 and have been remedied over there.